packages/utils-debugging-protocol-common/src/debugging-protocol-connector.ts

```diff
@@ -335,6 +335,17 @@ export class Connector implements IConnector {
 
         const eventName = `fetch::end::${suffix}` as 'fetch::end::*';
 
+        /*
+         * New versions of chrome are fetching the manifest
+         * automatically, so to prevent emit the event `fetch::end::manifest`
+         * twice, we need to ignore the event here.
+         * The manifest is already captured and processed by
+         * `parser-manifest` when the element is detected in the DOM.
+         */
+        if (suffix === 'manifest') {
```

Comment:
I'm not fully convinced about this. This code path could be executed before the traversing in the future (because chrome changes how this is implemented or because we change our traversing implementation). Maybe we should check in both places if the manifest has already been fetched before doing anything? During the travers this should be done before we do the request to download it and thus saving some time and resources.

What do you think?